app-arch/fastjar/fastjar-0.98.recipe

```diff
@@ -5,8 +5,10 @@ FastJar is an attempt at creating a feature-for-feature copy of Sun's JDK's \
 in Java which makes it dog slow.  Since FastJar is written in C, it can create \
 the same .jar file as Sun's tool in a fraction of the time."
 HOMEPAGE="https://savannah.nongnu.org/projects/fastjar"
-COPYRIGHT="1999-2001 Bryan Burns, Cory Jon Hollingsworth, Matthias Klose (documentation)
-	2007-2009  Dalibor Topic"
+COPYRIGHT="
+	1999-2001 Bryan Burns, Cory Jon Hollingsworth, Matthias Klose (documentation)
+	2007-2009 Dalibor Topic
+	"
```

Comment:
This was correct the way it was before; COPYRIGHT blocks don't get extra spacing.